auditlog/models/rule.py

```diff
@@ -105,6 +105,8 @@ class AuditlogRule(models.Model):
     action_id = fields.Many2one(
         'ir.actions.act_window', string="Action",
         states={'subscribed': [('readonly', True)]})
+    capture_record = fields.Boolean("Capture Record",
+        help=("Select this if you want to keep track of Unlink Record"),)
```

Comment:
Not a blocker. Just make it common.
```suggestion
    capture_record = fields.Boolean(
        string="Capture Record",
        help="Select this if you want to keep track of unlinked record",
    )
```